src/main/java/org/minions/devfund/sergio/battleship_game/Ocean.java

```diff
@@ -0,0 +1,53 @@
+package org.minions.devfund.sergio.battleship_game;
+
+import java.util.Random;
+
+public class Ocean {
+
+    private int shots = 0;
+    private int hitCount = 0;
+    private int sunkShips = 0;
+
+    public Ocean() {
+
+    }
+
+    public void placeAllShipsRandomly() {
+
+        Random random = new Random();
+        Ship[] ships = new Ship[13];
+    }
+
+    public boolean isFull(int row, int column) {
+        return true;
+    }
+
+    public boolean shootAt(int row, int column) {
+        this.shots++;
+        return false;
+    }
+
+//    public Ship[][] getShipArray() {
+//        return ships;
+//    }
+//
+//    public void setShips(Ship[][] ships) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)